packages/modelence/src/db/client.ts

```diff
@@ -38,9 +45,77 @@ export function getClient() {
   return client;
 }
 
-// export async function closeConnection() {
-//   if (client) {
-//     await client.close();
-//     client = null;
-//   }
-// }
+export async function closeConnection() {
+  if (healthCheckInterval) {
+    clearInterval(healthCheckInterval);
+    healthCheckInterval = null;
+  }
+  
+  if (client) {
+    await client.close();
+    client = null;
+  }
+  
+  reconnecting = false;
+}
+
+async function attemptReconnect() {
+  if (reconnecting) return;
+  reconnecting = true;
+
+  let retries = 0;
+  const maxRetries = 10;
+  
+  while (retries < maxRetries) {
+    try {
+      retries++;
+      console.log(`Attempting MongoDB reconnect (#${retries}/${maxRetries})...`);
```

Comment:
Could you replace `Attempting MongoDB reconnect` with `Reconnecting to MongoDB`?